Source/JavaScriptCore/jsc.cpp

```diff
@@ -844,8 +844,37 @@ static URL currentWorkingDirectory()
     return URL::fileURLWithFileSystemPath(directoryString);
 }
 
-static URL absolutePath(const String& fileName)
+static bool isAbsolutePath(StringView path)
 {
+#if OS(WINDOWS)
+    // Just look for local drives like C:\.
+    return path.length() > 2 && isASCIIAlpha(path[0]) && path[1] == ':' && (path[2] == '\\' || path[2] == '/');
+#else
+    return path.startsWith('/');
```

Comment:
Since it is used for module specifier too, I think / root path should be supported too in Windows. And in that case, we should make a path from the current path's drive.

Reply:
Hmm, are you sure? It seems like other engines still treat that as a relative path on Windows:

test.js
```
import result from '/GitHub/WebKit/test-module.js';
print(result);
```

Result:
```
PS C:\GitHub\WebKit> ~\.jsvu\v8.cmd --module test.js
undefined:0: Error: d8: Error reading  module from C:/GitHub/WebKit//GitHub/WebKit/test-module.js
    imported by C:/GitHub/WebKit/test.js

PS C:\GitHub\WebKit> ~\.jsvu\sm.cmd --module test.js
Error: can't open C:\GitHub\WebKit\\GitHub\WebKit\test-module.js: No such file or directory
Stack:
```